content/post/writing-a-book-remotely.md

```diff
@@ -0,0 +1,576 @@
+
+Furthermore, everyone had a different understanding on when to use a *bold*, _italic_ statement or when to use an info box. This was something we wanted to unify across the book.
+
+Next, we also replaced some of our architecture and information diagrams with nice-looking [Excalidraw](https://excalidraw.com/) diagrams:
+
+![Stratospheric Excalidraw Image Example](/img/stratospheric-ebook/stratospheric-excalidraw-example.png#center "Stratospheric Excalidraw Image Example")
+
+These finalization parts took almost two month as we also read the eBook from start to finish several times. We took over the proofreading and did not hire someone to do this.
+
+In total, we ended up with 450 pages of high-value content about Spring Boot and AWS.
+
+The version `1.0` should be both feature complete as well as polished. We knew that our eBook journey doesn't stop here but wanted to finally move the completness needle on Leanpub to 100%.
+
+As the finalization of the manuscript went closer, we thought about the best possible way to announce and release version 1.0.
+
+We were invited to the [German AWS Podcast](https://german-aws-podcast.libsyn.com/33-von-null-auf-produktion-mit-spring-boot-und-aws) from Dennis Traub (AWS Developer Advocate) and could talk about our own AWS cloud journey and advertise the eBook a bit.
+
+On top of this podcast, we decided to organize a release party and invited both the existing eBook readers and other interested persons from our audience.
+
+For this release party we prepared a one-hour agenda where we talked about the content of the book, showcased the sample application and the CI/CD workflow and had some time left for a Q&A session.
+
+From the ~50 accepted invitations, twenty people showed up on a Sunday afternoon to celebrate the release with us.
+
+After this 50 minutes of content, we released the final version of Stratospheric together with the audience by hitting the "Publish Now" button in Leanpub _together_ and live in the call.
+
+{{< tweet user="rieckpil" id="1421811091152191488" >}}
+
+We recorded this release party for those that couldn't make it to the live event on the 1st of August 2021 and [uploaded the video on YouTube](https://www.youtube.com/watch?v=oaWEKvVXucU):
+
+{{< youtube oaWEKvVXucU >}}
+
+We were all super happy with the outcome of the release party and our progress. At that time we already had 500 readers and getting more and more attention.
+
+## Life After the 1.0 Release 🏗
+
+> A book is never finished; it's abandoned. (Gene Fowler)
+
+While still being euphoric about releasing finish the first version of this book within one year, we knew that our journey wouldn't stop here. Right after the release party, we all took some time off and focussed more on spreading the news about the release rather than planning what's next.
```

Comment:
```suggestion
While still being euphoric about releasing finish the first version of this book within one year, we knew that our journey wouldn't stop here. Right after the release party, we all took some time off and focused more on spreading the news about the release rather than planning what was next.
```